.github/actions/install-dependencies-and-plottr/action.yml

```diff
@@ -0,0 +1,13 @@
+name: "Install-dependencies-and-plottr"
+description: "Install plottr and its dependencies"
+runs:
+  using: "composite"
+  steps:
+    - name: Install-dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install -r requirements.txt
+        pip install -r test_requirements.txt
+        pip install pyqt5
```

Comment:
is there a reason pyqt5 is not in install_requirements in setup.py?

Reply:
Yes, the problems is that both pyside2 and pyqt5. Furthermore the conda package is named pyqt so if you try to install plottr into a conda env you would install a broken set of dependencies